shared_model/backend/protobuf/query_responses/proto_block_response.hpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_SHARED_MODEL_PROTO_BLOCK_RESPONSE_HPP
+#define IROHA_SHARED_MODEL_PROTO_BLOCK_RESPONSE_HPP
+
+#include "backend/protobuf/block.hpp"
+#include "backend/protobuf/common_objects/trivial_proto.hpp"
+#include "interfaces/query_responses/block_response.hpp"
+#include "responses.pb.h"
+#include "utils/lazy_initializer.hpp"
+#include "utils/reference_holder.hpp"
+
+namespace shared_model {
+  namespace proto {
+    class BlockResponse final
```

Comment:
We now separate declaration from the definition for proto classes. Please look at the proto commands and do the same.